spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/simple/SimpleProperties.java

```diff
@@ -34,7 +34,8 @@
 public class SimpleProperties {
 
 	/**
-	 * Enable publishing to the backend.
+	 * Enable in-memory metrics that aren't published anywhere (allows you to see
+	 * what metrics are collected in the metrics actuator endpoint).
 	 */
 	private boolean enabled;
```

Comment:
The default value here should be true. And we shouldn't have a property at all since it's not used anywhere in the code.